This commit breaks suse RPMs.

For some reason the process is installing the files into `libexec` instead of `lib` and they are not found when building the RPM.

<pre>
Processing files: singularity-3.0.2-399.g9a80e4c9.x86_64
error: File not found: /home/vagrant/rpmbuild/BUILDROOT/singularity-3.0.2-399.g9a80e4c9.x86_64/usr/lib/singularity/bin/starter-suid
error: Directory not found: /home/vagrant/rpmbuild/BUILDROOT/singularity-3.0.2-399.g9a80e4c9.x86_64/usr/lib/singularity
error: File not found: /home/vagrant/rpmbuild/BUILDROOT/singularity-3.0.2-399.g9a80e4c9.x86_64/usr/lib/singularity/bin/starter
error: File not found: /home/vagrant/rpmbuild/BUILDROOT/singularity-3.0.2-399.g9a80e4c9.x86_64/usr/lib/singularity/cni/*

RPM build errors:
    File not found: /home/vagrant/rpmbuild/BUILDROOT/singularity-3.0.2-399.g9a80e4c9.x86_64/usr/lib/singularity/bin/starter-suid
    Directory not found: /home/vagrant/rpmbuild/BUILDROOT/singularity-3.0.2-399.g9a80e4c9.x86_64/usr/lib/singularity
    File not found: /home/vagrant/rpmbuild/BUILDROOT/singularity-3.0.2-399.g9a80e4c9.x86_64/usr/lib/singularity/bin/starter
    File not found: /home/vagrant/rpmbuild/BUILDROOT/singularity-3.0.2-399.g9a80e4c9.x86_64/usr/lib/singularity/cni/*
make: *** [Makefile:479: rpm] Error 1
</pre>

--- a/.travis.yml
+++ b/.travis.yml
@@ -38,7 +38,7 @@ matrix:
         - sw_vers
         - go version
         - singularity version
-        - singularity
+        - singularity help
 
 services:
   - docker

--- a/.travis.yml
+++ b/.travis.yml
@@ -36,7 +36,18 @@ matrix:
         - sudo make -C builddir install
       script:
         - sw_vers
-        - go version
+        - singularity version
+        - singularity help
+    - os: osx
+      osx_image: xcode9.2
+      before_install:
+        - # pass
+      install:
+        - ./mconfig
+        - make -C builddir
+        - sudo make -C builddir install
+      script:
+        - sw_vers
         - singularity version
         - singularity help
 

--- a/.travis.yml
+++ b/.travis.yml
@@ -26,6 +26,19 @@ matrix:
       env: OS_TYPE=centos OS_VERSION=6
     - os: linux
       env: OS_TYPE=centos OS_VERSION=7
+    - os: osx
+      osx_image: xcode10.1
+      before_install:
+        - # pass
+      install:
+        - ./mconfig
+        - make -C builddir
+        - sudo make -C builddir install
+      script:
+        - sw_vers
+        - go version
+        - singularity version
+        - singularity
 
 services:
   - docker

